nodes/src/nodes/ml_sklearn/code.py

```
@@ -0,0 +1,47 @@
+
+# ------------------------------------------------------------------------------
+# PreProcessor: sklearn-based text inference class
+# All heavy imports are deferred — this file is imported only after
+# depends() has installed requirements.txt in beginGlobal().
+# ------------------------------------------------------------------------------
+
+
+class PreProcessor:
+    """Wraps a scikit-learn model/pipeline for text inference."""
+
+    def __init__(self, config: dict):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)